documentation/asciidoc/microcontrollers/debug-probe/installing-tools.adoc

```diff
@@ -1,66 +1,12 @@
 == Install tools
 
-To use the Debug Probe, install the following tools.
+To use the Debug Probe, OpenOCD and the GNU Project Debugger (GDB) are required. An Integrated Development Environment (IDE) may also be useful.
 
-=== Install OpenOCD
+On Raspberry Pi OS, most Linux variants, and Microsoft Windows, it is recommended to install our VS Code extension. This extension bundles OpenOCD, ARM toolchains, GDB, and register definitions for Pico-series microcontrollers.
```

Comment:
Also macOS ?

Reply:
Excluding x86_64 macOS, where the extension is currently missing OpenOCD

Reply:
So what do x86_64 macOS users have to do?

Reply:
Probably use the homebrew OpenOCD head, which should have RP2350 support? The extension doesn't support debugging on x86_64 macOS

Reply:
Seems to me like that needs to go in the getting-started guide and not here.

Reply:
I agree that the distinction between x86_64 macOS and aarch64 macOS deserves to live in the PDF documentation, but IMHO if we're mentioning "most Linux variants, and Microsoft Windows" on this page, we also ought to mention macOS here.